01_04_Damage_Types_and_Armor_Types.txt

```diff
@@ -4,81 +4,89 @@ Damage Types and Armor Types
 
 ===== Damage Types =====
 
-== Weapon Types ==
-
-Ballistic Weapons:
-    Ballistic weapons are straightforward: they are blocked by armor and ignore 
-shields. Any damage that defeats armor damages health.
-
-Plasma Weapons:
-    Plasma interacts with both armor and shields. Armor's effectiveness is 
-reduced by 1 APL when hit by plasma. Targets hit with plasma that pierces armor, 
-take (stackable) plasma burn. For Shields, Plasma deals half damage to the 
-shielding, plus inflicts plasma burn and any remaining damage on the wearer 
-(assuming their armor is pierced). 
-
-Laser Weapons:
-    Lasers are completely blocked by shields, dealing all of their damage to the 
-shielding. Lasers deal full damage to armored targets; If a player wearing 
-ballistic armor is hit with a laser, they receive damage as if they weren't 
-wearing any armor.
-
-== Fall Damage ==
-
-    Beyond weapon damage, there are also environmental threats to consider. An 
-important note is on fall damage. A player can fall up to their Dex/2 in 
-meters before needing to roll an acrobatics check. If the surface they are 
-landing on is somehow fouled, the GM may require the player to roll an 
-acrobatics check when landing from any height at their discretion. On a failed 
-roll, take damage equal to 2 times the height fallen. The player also incurs 
-one injury, depending on the severity of the fall. Some examples are below:
-
-* Sprained ankle(s)
-* Broken ankle(s)
-* Broken leg(s)
-* Broken arm(s) in addition to leg(s)
-* Broken everything
-
-== Other Damage Types ==
-
-Explosives:
-    Certain grenades and devices deal Explosive damage, which ignores armor and 
-shields (explosive damage is essentially a crushing force). Though they may seem 
+== Common Damage Types ==
+
+Ballistic Damage:
+	Ballistic damage results from the impact and/or penetration of solid 
+projectiles, causing physical damage and secondary shock effects to living and 
+mechanical systems.
+    
+	Ballistic damage is dealt first to armor, and then to health. It 
+bypasses shields.
+
+Plasma Damage:
+	Plasma damage is dealt by energy transfer to the target from a high-energy 
+plasma projectile and/or spray. 
```

Comment:
nonblocking: this _may_ be a little technical, and doesn't add much flavor. While you and I had thermodynamics courses in college and we understand the damage dealt by heat and EM energy transfer, this doesn't really make any more of an explination than "plasma damage happens when you hit things with plasma". It's still better than having no prose whatsoever.